tox_extension_messages.c

```diff
@@ -35,6 +46,11 @@ struct ToxExtensionMessages {
 	tox_extension_messages_receipt_cb receipt_cb;
 	tox_extension_messages_negotiate_cb negotiated_cb;
 	void *userdata;
+	bool started_negotiation;
```

Comment:
I think we should remove this and just add our message size to the registration function. It looks like we don't support changing this after negotiation has started and it seems likely that a client will accidentally race this.

Reply:
Agreed, makes sense. Was just doing it that way to max the default val easier to use, but the added code and possible race definitely isn't worth it.